docs/products/opensearch/howto/upgrade-to-opensearch.rst

```diff
@@ -43,3 +43,58 @@ Extra information for Terraform users
 -------------------------------------
 
 If you manage your infrastructure with our :doc:`Terraform provider </docs/tools/terraform>` then please note that you will need to upgrade the provider to the latest release in order to have support for OpenSearch.
+
```

Comment:
When migrating through terraform, the steps described at the top of the file are not relevant, right? 

I feel that the terraform information stands on its own and for readers will be easier to understand it if it is put in a separate file. Can we move it to upgrade-to-opensearch-terraform.rst (or similar) and add a proper header "Upgrade to OpenSearch with Terraform"?

Reply:
Very true. I can move this to a separate doc.